fossunited/chapters/doctype/foss_chapter_event/foss_chapter_event.py

```diff
@@ -104,8 +104,30 @@ def copy_team_members(self):
             )
 
     def before_save(self):
+        if self.has_value_changed("status"):
+            self.update_published_status()
         self.set_route()
 
+    def update_published_status(self):
+        if self.status == "Draft" or self.status == "Cancelled":
+            self.is_published = 0
+            return
+
+        if self.status == "Live" or self.status == "Approved":
```

Comment:
Why `Approved` and `Live` together ? - We would only need `Live` or either one ?

Reply:
Because we have current events with Approved status. Will remove approved in future, it is here right now so that existing stuff do not break

Reply:
Considering we have only 4 live events - is there a possibility we could manually changed it in the desk soon after deploy?